tools/commit_slider/utils/preprocess/replace.py

```diff
@@ -1,12 +1,33 @@
 import re
 import fileinput
+import os
+from utils.helpers import CfgError
 
-
-def replace(cfg, commit):
+def replace(cfg, commit=None):
     prepCfg = cfg["runConfig"]["preprocess"]
     filePath = prepCfg["file"]
+    curDir = os.path.abspath(os.path.join(
+        os.path.dirname(__file__), "../../../../"))
+    filePath = os.path.join(curDir, filePath)
     pattern = prepCfg["pattern"]
     replacement = ''
+    if "replacement" in prepCfg:
+        replacement = prepCfg["replacement"]
     for line in fileinput.input(filePath, inplace=True):
         newLine = re.sub(pattern, r'{}'.format(replacement), line, flags=0)
         print(newLine, end='')
+
+# example: python3 -m commit_slider -u replace -file src/plugins/intel_cpu/src/graph.cpp  -pattern "\!node->isDynamicNode\(\)\ &&\ \!node->isExecutable\(\)\ &&\ \!node->isInPlace\(\)" -replacement "false"
```

Comment:
From the first glance, this "replacement" preprocess is an analogue of git patch applying. In case of git patch, the only thing we need is to specify the path to the patch. In comparison with the current approach, this would cost only one argument instead of 3

Do you think this preprocess can be optimized in this way?